### ISSUE TITLE
Metrics Fail to Report because the Sidekiq Enterprise Senate Leader method is incorrect

### ISSUE THREAD

**Issue by imightbeinatree** [fastly/sidekiq-prometheus#33]
Some but not all metrics stopped working with the 1.8.1 release. Specifically report_global_metrics and reported_redis_metrics in PeriodicMetrics have stopped working when combined with the latest versions of sidekiq-enterprise due to a bug in my previous sidekiq-enterprise related update which was accepted and released as 1.8.1.

I have worked through the issue and monkey patched the gem from our private metrics collection gem which uses sidekiq_prometheus.  I have now been running this without issue across our own applications on a variety of sidekiq versions, some with enterprise and some without.  The metrics that stopped with the 1.8.1 release are working again.

I have opened a PR [#32](https://github.com/fastly/sidekiq-prometheus/pull/32) that does fix this issue in sidekiq_prometheus, however Sidekiq 7 released just a few days after I started this work.  Sidekiq 7 causes further issues that I am unable to address at this time so I changed the dependency requirement.  I wanted to open this issue to start a discussion and see how you felt about these changes and what we could do to get this update merged.

**Comment by hiddewie** [comment]
Bug confirmed for :
```
sidekiq 6.5.8        
sidekiq-pro 5.5.2    
sidekiq-ent 2.5.1    
```

**Comment by leklund** [comment]
@imightbeinatree Thanks for the fix! I've merged #32 and released v1.8.2.
cc: @hiddewie

**Comment by hiddewie** [comment]
Thanks for the quick fix & merge! 🎉
